packages/yoga/src/Card/web/PlanCard/List.jsx

```diff
@@ -31,11 +32,20 @@ const Item = styled.li`
   }
 `;
 
-const Wrapper = styled.div`
+const Wrapper = styled(Box)`
   display: flex;
   min-width: 0;
   align-items: center;
 
+  ${props =>
```

Comment:
Clicks can also be a link, Should we add that option? Example:
If the component receives a `href` prop, we can assume element should be a `<a>` tag..

WDYT?

Reply:
Also, beware that `button` should have the "type" attribute. (Maybe we can assume always `type="button"`).

A helpful article: https://www.builder.io/blog/buttons

Reply:
> Clicks can also be a link, Should we add that option? Example: If the component receives a `href` prop, we can assume element should be a `<a>` tag..
> 
> WDYT?

I have discussed with the team, and for now, links are out of scope, at least within the context of Gympass.

Reply:
> Also, beware that `button` should have the "type" attribute. (Maybe we can assume always `type="button"`).
> 
> A helpful article: https://www.builder.io/blog/buttons

Nice article! I completely agree. I added the button type in this commit: 5845523425325a78f71ac90ce709fe97d41866d7.


@MicaelRodrigues could you review it again please?